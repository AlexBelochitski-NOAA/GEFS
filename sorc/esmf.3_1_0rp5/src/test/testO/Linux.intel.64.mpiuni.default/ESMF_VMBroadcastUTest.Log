20190610 112234.082746 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112234.083010 INFO     PET0  Beginning Test, file ESMF_VMBroadcastUTest.F90, line          74
20190610 112234.083028 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112234.083063 INFO     PET0  PASS  Broadcasting  data Test, ESMF_VMBroadcastUTest.F90, line         128
20190610 112234.083080 INFO     PET0  PASS  Verify local data after broadcast Test, ESMF_VMBroadcastUTest.F90, line         141
20190610 112234.085422 INFO     PET0  PASS  Broadcasting local data Test, ESMF_VMBroadcastUTest.F90, line         152
20190610 112234.085497 INFO     PET0  PASS  Verify local data after Broadcast Test, ESMF_VMBroadcastUTest.F90, line         168
20190610 112234.085527 INFO     PET0  PASS  Broadcasting local data Test, ESMF_VMBroadcastUTest.F90, line         179
20190610 112234.085585 INFO     PET0  PASS  Verify local data after Broadcast Test, ESMF_VMBroadcastUTest.F90, line         193
20190610 112234.085596 INFO     PET0  PASS  Broadcasting local data Test, ESMF_VMBroadcastUTest.F90, line         204
20190610 112234.085684 INFO     PET0  PASS  Verify local data after broadcast Test, ESMF_VMBroadcastUTest.F90, line         224
20190610 112234.085699 INFO     PET0  Number of failed tests:           0
20190610 112234.085706 INFO     PET0  Ending Test, file ESMF_VMBroadcastUTest.F90, line         226
