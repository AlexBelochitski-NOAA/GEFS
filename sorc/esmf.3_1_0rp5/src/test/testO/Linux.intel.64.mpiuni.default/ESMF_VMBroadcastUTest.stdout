  Beginning Test, file ESMF_VMBroadcastUTest.F90, line          74
  NUMBER_OF_PROCESSORS           1
  PASS  Broadcasting  data Test, ESMF_VMBroadcastUTest.F90, line         128
           0  After Broadcast  LocalData is            1           2 ( Should be
             1           2  )
  PASS  Verify local data after broadcast Test, ESMF_VMBroadcastUTest.F90, line 
         141
  PASS  Broadcasting local data Test, ESMF_VMBroadcastUTest.F90, line         15
 2
           0 After Broadcast: R4_LocalData is    1.000000       2.000000     
 ( Should be    1.000000       2.000000      )
           0 After Broadcast r4_LocalData is    1.000000       2.000000    
  PASS  Verify local data after Broadcast Test, ESMF_VMBroadcastUTest.F90, line 
         168
  PASS  Broadcasting local data Test, ESMF_VMBroadcastUTest.F90, line         17
 9
           0 After Broadcast: R8_LocalData is    1.00000000000000     
   2.00000000000000       (Should be   1.00000000000000     
   2.00000000000000       )
  PASS  Verify local data after Broadcast Test, ESMF_VMBroadcastUTest.F90, line 
         193
  PASS  Broadcasting local data Test, ESMF_VMBroadcastUTest.F90, line         20
 4
           0 after broadcast: Local_Logical(1) is False
           0 after broadcast: Local_Logical(2) is True
           0 After broadcast: logical_soln(1) is False
           0 After broadcast: logical_soln(2) is True
  PASS  Verify local data after broadcast Test, ESMF_VMBroadcastUTest.F90, line 
         224
  Number of failed tests:           0
  Ending Test, file ESMF_VMBroadcastUTest.F90, line         226
